lessons/Data-Science/ai-linear-regression/langs/en-us/pages/alvinn-training.adoc

```diff
@@ -0,0 +1,61 @@
+= Supervised Machine Learning: Training ALVINN
+
+@n Imagine that ALVINN has trained on the 15 images collected during a half minute of driving. How do you predict its best guesses about the steering angle will be? Explain. @fitb{}{}
+
+@fitb{}{@ifsoln{Given that 15 images is a relatively small amount, ALVINN's steering angle guesses will be essentially random.}}
+
+@fitb{}{@ifsoln{A function built on just 15 ordered pairs is not going to have a high level of accuracy.}}
+
+@fitb{}{}
+
+
+@n Respond to the questions below, considering how each variation in training will influence ALVINN's performance.
```

Comment:
These all seem different enough that they should be different numbers, rather than sub-questions of question 2. If they were all different questions about a model after a single training, that would be different.

Reply:
Ugh. They used to all by their own numbers. Shriram thought that was confusing and asked me to change them to sub-questions.